src/components/SourceSearch.js

```diff
@@ -20,15 +20,15 @@ function searchResults(sources) {
   function getSourcePath(source) {
     const { path, href } = parseURL(source.get("url"));
     // for URLs like "about:home" the path is null so we pass the full href
-    return endTruncateStr((path || href), 50);
+    return (path || href);
   }
 
   return sources.valueSeq()
     .filter(source => !isPretty(source.toJS()) && source.get("url"))
     .map(source => ({
       value: getSourcePath(source),
       title: getSourcePath(source).split("/").pop(),
```

Comment:
is it weird to only show the title if the subtitle is `> 50`? 

I'm asking because the  tooltip is a little bit of noise when you pause over the item. 

Happy to keep it in all cases if that's the *proper* ux :)